changelog

```diff
@@ -14,6 +18,27 @@ Version 0.9.15:
       * The luxury accommodations outfit now has a proper plural form. (@quyykk)
       * "Defend New Tibet" now has a fail dialog. (@Terin)
       * Some Coalition missions are no longer offered on stations. (@Arachi-Lover)
+      * Fixed the failure conditions of Earth Day missions to properly fail once Earth Day has passed. (@DownsB)
+      * Corrected various uses of concrete vs cement. (@Zitchas)
+      * Changed various mission cargo to lowercase. (@Anarchist2)
+      * Converted a 16-bit linear sprite that rendered strangely to 8-bit sRGB. (@Anarchist2)
+      * Prevented some Hai missions from offering on uninhabited planets. (@Mach565)
+      * Fixed some ships that had more ammo than they could actually hold. (@Hecter94)
+      * Fixed the blending mode on the Remnant afterburner graphics. (@Anarchist)
+      * The first Pact Recon mission is now marked as minor. (@Anarchist2)
+      * Fixed a Punisher variant with a missing hyperdrive. (@quyykk)
+      * "FW Pug 1C Return" now doesn't trigger until after the player has visited the waypoint in "FW Pug 1C". (@Amazinite)
+      * Fixed the personality of some Remnant NPCs in missions that should have been staying. (@alextd, @quyykk)
+      * Enforced some writing style changes across the whole game, such as proper nouns which end in "s" becoming "s'" when possessive as opposed to "s's" and action choices always being in first person. (@Amazinite, @Anarchist2, @Zitchas)
+      * Fixed some Remnant ship variants that had accidentally been given extra keystones. (@pilover100)
+      * The Hunted mission that sends bounty hunters after the player now more accurately measures the player's fleet strength. (@Amazinite)
+      * Fixed reputation changes to the Hai from missions to also apply to all the new Hai governments. (@Hurleveur)
+      * The Unfettered Tribute jobs now mention both possible missing objectives in the on visit dialog. (@Hurleveur)
+      * The Coalition Finisher now plays a sound effect when it impacts a target. (@Arachi-Lover)
+      * The Pre-Hai first contact mission for the Wanderers no longer offers if you've met the Unfettered, but not the Hai. (@Arachi-Lover)
+      * Fixed errors in various ship images. (@leklachu)
+      * Fixed the lighting on a new Korath ship sprite. (@Anarchist2)
```

Comment:
Should we list bug fixes for content added in the same update? From a holistic (that's definitely not the right word) viewpoint, there isn't any difference between a bug being noticed and fixed when reviewing a PR and a bug being fixed in a follow up PR.